Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -676,15 +676,27 @@ class CalculatedStyleValue : public StyleValue {
     ResolvedType resolved_type() const { return m_resolved_type; }
     NonnullOwnPtr<CalcSum> const& expression() const { return m_expression; }
 
+    bool resolves_to_angle() const { return m_resolved_type == ResolvedType::Angle; }
     Optional<Angle> resolve_angle() const;
-    Optional<AnglePercentage> resolve_angle_percentage(Angle const& percentage_basis) const;
+    Optional<Angle> resolve_angle_percentage(Angle const& percentage_basis) const;
```

Comment:
These `AnglePercentage` -> `Angle` changes below to the previous commit :^)

Reply:
Oops, thanks for pointing that out!